{
  "comments": [
    {
      "key": {
        "uuid": "a9ab3f78_9e907a33",
        "filename": "/COMMIT_MSG",
        "patchSetId": 10
      },
      "lineNbr": 9,
      "author": {
        "id": 1084448
      },
      "writtenOn": "2018-10-02T14:09:21Z",
      "side": 1,
      "message": "Nit: Same mistake here.",
      "range": {
        "startLine": 9,
        "startChar": 52,
        "endLine": 9,
        "endChar": 53
      },
      "revId": "aede302fa8728d25305df9c2eb8ad695eb689da6",
      "serverId": "85c56323-6fa9-3386-8a01-6480fb634889",
      "unresolved": true
    }
  ]
}